src/tests/unit/tests/electron/views/root-container/components/platform-body-class-modifier.test.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { DocumentManipulator } from 'common/document-manipulator';
+import {
+    PlatformBodyClassModifier,
+    PlatformBodyClassModifierDeps,
+} from 'electron/views/root-container/components/platform-body-class-modifier';
+import { PlatformInfo } from 'electron/window-management/platform-info';
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { IMock, Mock } from 'typemoq';
+
+describe(PlatformInfo, () => {
+    let mockPlatformInfo: IMock<PlatformInfo>;
+    let mockDocumentManipulator: IMock<DocumentManipulator>;
+    let mockDeps: PlatformBodyClassModifierDeps;
+
+    beforeEach(() => {
+        mockPlatformInfo = Mock.ofType<PlatformInfo>();
+        mockDocumentManipulator = Mock.ofType<DocumentManipulator>();
+        mockDeps = {
+            platformInfo: mockPlatformInfo.object,
+            documentManipulator: mockDocumentManipulator.object,
+        };
+    });
+
+    it('Renders as null for non-mac', () => {
+        mockPlatformInfo.setup(m => m.isMac()).returns(() => false);
+
+        const testSubject = shallow(<PlatformBodyClassModifier deps={mockDeps} />);
+
+        expect(testSubject.getElement()).toBeNull();
+    });
+
+    it('Renders per snapshot for mac', () => {
+        mockPlatformInfo.setup(m => m.isMac()).returns(() => true);
+
+        const testSubject = shallow(<PlatformBodyClassModifier deps={mockDeps} />);
+
+        expect(testSubject.debug()).toMatchSnapshot();
```

Comment:
the snapshot for this test seems interesting to me... I'm guessing is because the use of `debug()` on this line. Does using `getElement()` get us `documentManipulator={...}` (or similar)?? Is that the reason you used `debug()` instead of `getElement()`?